x/emissions/keeper/keeper.go

```diff
@@ -484,68 +484,76 @@ func (k *Keeper) SetOneInForecasterSelfNetworkRegret(ctx context.Context, topicI
 
 // Returns the regret of a worker from comparing loss of worker relative to loss of other inferers
```

Comment:
In the comment we mix/match worker and inferers, same terminology should be maintained  to avoid confusion.